src/unix/os390.c

```diff
@@ -624,6 +624,7 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
     }
 
     address->is_internal = flg.ifr_flags & IFF_LOOPBACK ? 1 : 0;
+    memset(address->phys_addr, 0, sizeof(address->phys_addr));
```

Comment:
Added the `memset()` in `uv__interface_addresses_v6()`, thanks!

Do you have any idea why [this same `netmask` logic](https://github.com/libuv/libuv/blob/95b5df7ae9692ddce322a5d2c072e28cedfa5fcf/src/unix/aix-common.c#L261-L269) wasn't ported to zos?

Reply:
After some change history archeology I think it is because this file was based on the `src/unix/aix.c` which used to have the same TODO comment until https://github.com/libuv/libuv/pull/1410. 

https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.2.0/com.ibm.zos.v2r2.bpxbd00/ioctl.htm says that `SIOCGIFNETMASK` is available (albeit `valid only for the AF_INET domain`) so it should be possible to port the logic over (at least for the AF_INET case).